### DISCUSSION TITLE
Is greenclip supposed to support wayland?

### DISCUSSION THREAD

**Discussion by ValJed** [erebe/greenclip#161] [Q&A] [answered]
Hello and thanks for this great tool!!

I recently switched from I3 to Hyprland and  it looks like greenclip stopped  working (not registering copied stuff or pasting  when selected from list).
So just wondering, maybe it's just not compatible with wayland?
Thank you!

**Comment by tuurep** [comment, marked as answer]
I remember there being an issue in the past where the maintainer said Wayland support would require a whole rewrite/starting as a new project altogether (if I do remember correctly)

So sadly it's not Wayland-compatible

**Comment by ValJed** [comment]
Thanks for this answer.
I switched to cliphist[https://github.com/sentriz/cliphist] in combination with `rofi-wayland`.
Works well.
